### ISSUE TITLE
Can we adjust this theme for Gitea v1.21.11?

### ISSUE THREAD

**Issue by bioinformatist** [TylerByte666/gitea-matrix-template#17]
This matrix theme looks excellent!

**Comment by TylerByte666** [comment]
Good day @bioinformatist! First off let me say I do apologize for the late reply. We are very busy at the moment.

Thank you so much for your positive feedback. I will be working on this repo in the next coming weeks to get it up to date. I will let you know as soon as we are done. 

I will be setting up a more frequent update plan too.

Thank you again ❤️

**Comment by bioinformatist** [comment]
Thanks for reply! Maybe making it compatible with the latest Gitea would be better, thanks in advance!

**Comment by TylerByte666** [comment]
Yes, that's the plan. we will then catch up the older versions after 🚀
